test/iso15118/session/feedback.cpp

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Pionix GmbH and Contributors to EVerest
+#include <catch2/catch_test_macros.hpp>
+
+#include <iso15118/session/feedback.hpp>
+
+using namespace iso15118::session;
+
+SCENARIO("Feedback Tests") {
```

Comment:
Finally, *easy-to-read* unit tests :+1: 
Btw. I would have created a struct like
```
struct FeedbackResults {
  feedback::Signal signal;
  feedback::DcChargeTarget dc_charge_target;
  ...
}
```
and then only use an instance of that type like `FeedbackResults feedback_results` and use `feedback_results.signal` instead of signal and so on.  This increases cohesion and readability in my point of view.

Reply:
Good idea with the struct. I'll change that.